includes/polling/cisco-sla.inc.php

```diff
@@ -42,12 +43,13 @@
 
         $fields = array(
             'rtt' => $rtt,
+            'opstatus' => $opstatus,
```

Comment:
Not defined for RRD (and rrd are immutable)

Reply:
You'll need to create an additional RRD file to hold this new data.

Reply:
@tamirhad Any response to this?